apps/i18n/common/en_us.json

```diff
@@ -824,6 +824,7 @@
   "keyValuePairLink": "Key/value pairs",
   "languages": "Languages",
   "lastEdited": "Last Edited",
+  "lastPublished": "Last Published",
```

Comment:
Just to be clear, this is the last updated date?

Reply:
yes! we've been referring to libraries' as being published/re-published, so i kept that language here